Tests/DataFixtures/Dumper/YamlDataDumperTest.php

```diff
@@ -40,15 +40,21 @@ public function testYamlDump()
 
         $loader = new YamlDataDumper(__DIR__.'/../../Fixtures/DataFixtures/Loader', array());
         $loader->dump($filename);
+        /**
+         * PHP 8.1.0 introduced “proper data-type support” in PDO SQLite.
+         * Before that, all values were returned as strings even if they were really other types.
+         * That’s not a problem for us in production because, obviously, we don’t use SQLite.
+         */
+        $id = (float)phpversion() >= 8.1 ? 1 : "'1'";
```

Comment:
🔧 Please don't cast version strings to float numbers. This is prone to break at some point.
```suggestion
        $id = version_compare(phpversion(), '8.1', '>=') ? 1 : "'1'";
```